taxi/src/main/java/com/epam/training/taxi/account/UnregisteredAccount.java

```diff
@@ -0,0 +1,19 @@
+package com.epam.training.taxi.account;
+
+public final class UnregisteredAccount implements Account {
```

Comment:
I like what you did here - using polymorphic classes over conditions often works well.